src/pages/Oauth/Logout.tsx

```diff
@@ -0,0 +1,37 @@
+import KyberOauth2 from '@kybernetwork/oauth2'
+import { Trans } from '@lingui/macro'
+import { useEffect } from 'react'
+
+import Dots from 'components/Dots'
+import { ENV_KEY } from 'constants/env'
+import useParsedQueryString from 'hooks/useParsedQueryString'
+import { PageContainer } from 'pages/Oauth/styled'
+
+function Logout() {
+  const { logout_challenge } = useParsedQueryString<{ logout_challenge: string }>()
+
+  useEffect(() => {
+    if (!logout_challenge) return
+    KyberOauth2.initialize({ mode: ENV_KEY })
+    KyberOauth2.oauthUi
+      .acceptLogout(logout_challenge)
+      .then(data => {
+        console.debug('logout resp', data)
+      })
+      .catch(err => {
+        console.debug('err logout', err)
```

Comment:
Just suggest better log format
```suggestion
        console.debug('Oauth logout error', { error: err })
```